pkg/docker/docker.css

```diff
@@ -317,6 +317,18 @@ table > tbody > tr > .cell-buttons {
     height: 25px;
 }
 
+.container-details-name {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    max-width: calc(100% - 350px);;
```

Comment:
The 350px for all other columns seem a little arbitrary. How would that work on narrow layouts like mobile? Perhaps something like "40%" might work better? Bikeshedding of course, maybe @garrett has an idea about best practice here?

Thanks for addressing this!

Reply:
> Perhaps something like "40%" might work better?

My first thought was the same, but there is problem, that the buttons may take more than 40% of the space. See attached image. And this 350px is only for the two texts in the container details page.
![40p](https://user-images.githubusercontent.com/12330670/51462989-52859f00-1d62-11e9-9c6e-071a01115b1f.png)